app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt

```diff
@@ -187,12 +188,12 @@ private fun PlantDetailsContent(
     isPlanted: Boolean,
     onNamePosition: (Float) -> Unit,
     callbacks: PlantDetailsCallbacks,
-    transitionState: TransitionState
+    contentAlpha: State<Float>
```

Comment:
I was trying to avoid passing the Float down to avoid having to read it from the parent composable. But what I can do is passing `() -> Float` that's equivalent to `State<Float>` behavior. Thanks!